server/src/main/java/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -101,9 +103,14 @@
  *         ha.retry.wait | time to wait before retrying the work item | seconds | 120 || || stop.retry.wait | time to wait
  *         before retrying the stop | seconds | 120 || * }
  **/
-public class HighAvailabilityManagerImpl extends ManagerBase implements HighAvailabilityManager, ClusterManagerListener {
+public class HighAvailabilityManagerImpl extends ManagerBase implements HighAvailabilityManager, ClusterManagerListener, Configurable {
 
     protected static final Logger s_logger = Logger.getLogger(HighAvailabilityManagerImpl.class);
+    private ConfigKey<Integer> MaxRetries = new ConfigKey<>("Advanced", Integer.class,
+            "max.retries","5",
```

Comment:
`max.retries` is too generic, can you rename this to `vm.ha.migration.max.retries` or something suitable?

Reply:
This is an old config file in the code but was not exposed to be configured. Renaming makes sense. Doing it.

Reply:
If the global setting already exists, then don't rename them (for backward compatibility reasons) or re-define them. If they are new settings rename them.

Reply:
These existed before. Leaving them as is.

Reply:
I checked 4.13, the global setting `max.retries` does not exist in it however in code it is indeed referenced. Okay to leave as is, in that case.

Reply:
Yes. This was a bug because max.retries wasn't accessible via the API. We have fixed that here.

Reply:
@anuragaw @rhtyd 
sorry I have some late comments as this PR has been merged.

1) max.retries is not used in the past so I suggest to rename it before 4.14 release.
2) this config has scope=Cluster but cluster-wide configuration is never used in code.